javascript/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Use an official Node Docker image as the base
-FROM node:20.7.0
+FROM node:20.7.0-alpine
```

Comment:
I'm curious about this-- is alpine a necessity for this to work or is it an optimization to keep the image size smaller?

Reply:
20.7.0 size is about 1 GB, 20.7.0-alpine is only 181 MB, so a much slimmer image